ui/shared/components/panel/variants/FamilyVariantTags.jsx

```diff
@@ -262,6 +262,7 @@ const FamilyVariantTags = (
           <NoteContainer>
             {savedVariant && savedVariant.notes.map(note =>
               <VariantNoteField
+                showInLine
```

Comment:
I would actually move this prop directly into `VariantNoteField`, since you always want variant notes to have this prop

Reply:
Try passing the `noModal` prop as well